.github/workflows/rust.yml

```diff
@@ -27,9 +28,10 @@ jobs:
     container:
       image: docker://rust:1.62.0
     steps:
+      - uses: extractions/setup-just@aa5d15c144db4585980a44ebfdd2cf337c4f14cb
```

Comment:
Would it make sense to specify the just version here to make sure it's the same one we use in the devcontainer?

Reply:
Makes sense!